static/app/components/onboarding/gettingStartedDoc/sdkDocumentation.tsx

```diff
@@ -6,89 +6,6 @@ import {PlatformKey} from 'sentry/data/platformCategories';
 import type {Organization, PlatformIntegration, Project, ProjectKey} from 'sentry/types';
 import {useApiQuery} from 'sentry/utils/queryClient';
 
-// Documents already migrated from sentry-docs to main sentry repository
-export const migratedDocs = [
```

Comment:
after merging this PR the migration is going to be completed, therefore we will no longer need this list